File: Java/app/src/main/java/kz/sgq/fs_tiktaktoe/utilit/AnswerField.java
package kz.sgq.fs_tiktaktoe.utilit;

public class AnswerField {
    private static int[][] answerArr = {{0, 1, 2},
            {0, 3, 6},
            {0, 4, 8},
            {1, 4, 7},
            {2, 4, 6},
            {2, 5, 8},
            {3, 4, 5},
            {6, 7, 8}};

    public static int getAnswer(int i, int j) {
        return answerArr[i][j];
    }
}

File: Kotlin/app/src/main/java/kz/sgq/kotlintictactoe/utils/AnswerField.kt
package kz.sgq.kotlintictactoe.utils

object AnswerField {
    val answerArr = arrayOf(intArrayOf(0, 1, 2),
            intArrayOf(0, 3, 6),
            intArrayOf(0, 4, 8),
            intArrayOf(1, 4, 7),
            intArrayOf(2, 4, 6),
            intArrayOf(2, 5, 8),
            intArrayOf(3, 4, 5),
            intArrayOf(6, 7, 8))

    fun getAnswer(i: Int, j: Int) = answerArr[i][j]
}